### ISSUE TITLE
Sync up demo docs, with bz-database readme

### ISSUE THREAD

**Issue by PicoCreator** [unee-t/frontend#203]
Readme for bz-database : https://github.com/unee-t/bz-database
Readme for demo : https://documentation.unee-t.com/2018/03/01/introduction-to-the-demo-environment/

Certain information, such as role of user is incredibly useful for the setup and usage of bz-database, and perhaps other info as well.

**Comment by franck-boullier** [comment]
Not sure if this helps but just in case:
Roles and their corresponding role_type_id in the BZ database.
This is maintained in the BZ table 'ut_role_types'
* Tenant: 1
* Landlord: 2
* Contractor: 3
* Management company: 4
* Agent: 5

Not sure what else you need as the roles are not granting any specific permissions or privileges other than people in similar role being grouped together.

That being said, I agree that we need to update the readme file to reflect this and be clearer...

**Comment by franck-boullier** [comment]
Readme for demo : https://documentation.unee-t.com/2018/03/01/introduction-to-the-demo-environment/

In the future, this is where information about the demo environment will be maintained and updated.

**Comment by kaihendry** [comment]
Would prefer it in Github, ideally in a succinct well maintained README.md

Media dissonance is a thing!

**Comment by franck-boullier** [comment]
> Media dissonance is a thing!

I totally agree with that! IMO we should make sure the media with the widest audience is always up to date

> ideally in a succinct well maintained README.md

That'll very succinct and easy to maintain: my idea is to simply add a link to the relevant part of the documentation in the README.md file :stuck_out_tongue:
